extension/httpforwarder/config.go

```diff
@@ -0,0 +1,29 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package httpforwarder
+
+import (
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config defines configuration for http forwarder extension.
+type Config struct {
+	configmodels.ExtensionSettings `mapstructure:",squash"`
+	confighttp.HTTPServerSettings  `mapstructure:",squash"`
```

Comment:
I would also name this "Incoming" and the client "Outgoing", what do you think? same in config probably to make it clear for user.

Reply:
Updated.